.github/workflows/ci-wheels.yml

```
@@ -78,14 +98,22 @@ jobs:
         CIBW_MANYLINUX_X86_64_IMAGE: manylinux2014
         CIBW_BEFORE_BUILD_LINUX: yum install -y udunits2-devel
         CIBW_BEFORE_BUILD_MACOS: brew install udunits
+        CIBW_BEFORE_BUILD_WINDOWS: python -m pip install delvewheel
         CIBW_TEST_REQUIRES: pytest
         CIBW_TEST_COMMAND: >
           python -c 'import cf_units; print(f"cf-units v{cf_units.__version__}")' &&
           python -m pytest --pyargs cf_units
+        CIBW_TEST_COMMAND_WINDOWS: python -m pytest --pyargs cf_units
```

I imagine the original command was failing on Windows (as you mention in #556) due to the `&&` operator not being supported by the Windows shell (at least not in the version provided in the GHA runner).

To keep it consistent with the previous behaviour you could do this:

```suggestion
        CIBW_TEST_COMMAND_WINDOWS: >
          python -c 'import cf_units; print(f"cf-units v{cf_units.__version__}")
          if ($?) { python -m pytest --pyargs cf_units }
```

or seeing in this case running the tests isn't _really_ dependent on the previous command invocation, just simply running both commands independently is maybe acceptable?

```suggestion
        CIBW_TEST_COMMAND_WINDOWS: >
          python -c 'import cf_units; print(f"cf-units v{cf_units.__version__}")
          python -m pytest --pyargs cf_units
```